# confidence level to report objects as detected
ssd_confidence: 0.5

image_topic: /camera/color/image_raw
point_cloud_topic: /camera/depth_registered/points
camera_frame: panda_camera
base_frame: panda_link0
use_median: False
